src/strongbox/core.clj

```diff
@@ -374,6 +412,12 @@
 
 ;; selecting addons
 
+(defn-spec select-addons-search* nil?
+  "sets the selected list of addons to the given `selected-addons` for bulk operations like"
```

Comment:
"sets the selected list of addons in application state for installation in a later action"